core/lib/refinery/core/engine.rb

```diff
@@ -26,6 +26,35 @@ def refinery_inclusion!
         end
       end
 
+      def self.frontend_routes(&block)
+        @frontend_routes ||= []
+        unless @frontend_routes.include?(block)
```

Comment:
To clarify, I would have thought we'd use `@@frontend_routes` or `frontend_routes` (though the latter would not stick around).